pages/common/pcapfix.md

```diff
@@ -0,0 +1,24 @@
+# pcapfix
+
+> Prepair pcap and pcapng files.
+> More information: <https://f00l.de/pcapfix/>.
+
+- Repair a pcap/pcapng file::
+
+`pcapfix {{test.pcap}}`
+
+- Repair entire pcap/pcapng file:
+
+`pcapfix --deep-scan {{test.pcap}}`
+
+- Repair a pcap/pcapng file and write the repaired file to a specified file:
```

Comment:
```suggestion
- Specify the name of the repaired `pcap`/`pcapng` file:
```
It seems like this argument is required, so it should be in all examples.

Reply:
If no output file is explicitly specified, it is saved in a file with the `fixed_` prefix.